### ISSUE TITLE
RTMS explanation

### ISSUE THREAD

**Issue by ludoro** [SMTorg/smt#213]
Hello guys! First of all, really cool library. 

I have been working on Surrogates.jl, a Julia package that mostly has the same goal. 
I am trying to implement from this paper here: "A fast-prediction surrogate model for large datasets" which I believe it's related to this library.
However, I am not understanding how to build the matrix $F$, for each i-esim row I should evaluate the i-esim training point x at some splines, right? The number of columns is the degree of the splines? I am bit confused. Could you shed some light?

**Comment by relf** [comment]
Hi, thanks for the kind word. 
You're right the surrogate is related to this [paper](https://www.researchgate.net/profile/John_Hwang2/publication/322869949_A_fast-prediction_surrogate_model_for_large_datasets/links/5a737e03aca2720bc0dbc41f/A-fast-prediction-surrogate-model-for-large-datasets.pdf) (link broken in the doc) and well... as far as I am concerned I am not that familiar with this surrogate so I would suggest to refer to the implementation... but maybe you already did?!

**Comment by ludoro** [comment]
Yeah, I tried but without much success. I get stuck at page 6 of that paper, when defining the F function. I am not in a hurry so I can maybe wait for someone who is familiar with it :)
